content/influxdb3/core/query-data/execute-queries/influxdb-v3-api.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Use the InfluxDB v3 HTTP query API to query data
```

Comment:
Long title. The `seotitle` can keep the length, but I'd suggest shortening this.

```suggestion
title: Use the InfluxDB v3 HTTP query API
```